Release Notes/2015-09-01-cloud-platform-release-notes.md

```diff
@@ -57,9 +57,9 @@
 
 ### Ecosystem: New Blueprints (5)
 
-* __[Commvault](http://www.ctl.io/knowledge-base/ecosystem-partners/marketplace-guides/getting-started-with-commvault-server/)__ Commvault has integrated their enterprise backup technology software, Simpana, with the CenturyLink Cloud. In addition to installing the software, the CenturyLink Cloud Blueprint deployment include Commvault configuration best practices.  If you need to address the business challenges of data growth, data protection, end user protection or need a file-level backup solution that uses de-dupe technology, check out Commvault.  The Blueprints make it easy to deploy to your environment.
+* __[Commvault](https://www.ctl.io/knowledge-base/ecosystem-partners/marketplace-guides/getting-started-with-commvault-storage-blueprint/)__ Commvault has integrated their enterprise backup technology software, Simpana, with the CenturyLink Cloud. In addition to installing the software, the CenturyLink Cloud Blueprint deployment include Commvault configuration best practices.  If you need to address the business challenges of data growth, data protection, end user protection or need a file-level backup solution that uses de-dupe technology, check out Commvault.  The Blueprints make it easy to deploy to your environment.
 
-* __[Lagrange CloudMaestro](http://www.ctl.io/knowledge-base/ecosystem-partners/marketplace-guides/getting-started-with-lagrange-systems-cloudmaestro/)__ CloudMaestro is an Application Delivery Controller delivered with a Software-as-a-Service control plane for exceptional performance and availability in cloud infrastructure.  Deploying the CloudMaestro Agent Blueprint on your existing server in the CenturyLink cloud enables CloudMaestro to monitor, alert, replace and scale your application in response to traffic or unforeseen downtime.
+* __[Lagrange CloudMaestro](https://www.ctl.io/knowledge-base/ecosystem-partners/marketplace-guides/getting-started-with-lagrange-systems-cloudmaestro-blueprint/)__ CloudMaestro is an Application Delivery Controller delivered with a Software-as-a-Service control plane for exceptional performance and availability in cloud infrastructure.  Deploying the CloudMaestro Agent Blueprint on your existing server in the CenturyLink cloud enables CloudMaestro to monitor, alert, replace and scale your application in response to traffic or unforeseen downtime.
```

Comment:
sorry again but why are these fully qualified links? shouldn't this just be `../ecosystem-partners/marketplace-guides/getting-started-with-lagrange-systems-cloudmaestro-blueprint.md` ?